src/python/grpcio_tests/tests_py3_only/interop/xds_interop_client.py

```diff
@@ -129,6 +148,23 @@ def GetClientStats(self, request: messages_pb2.LoadBalancerStatsRequest,
         logger.info("Returning stats response: {}".format(response))
         return response
 
+    def GetClientAccumulatedStats(
+            self, request: messages_pb2.LoadBalancerAccumulatedStatsRequest,
+            context: grpc.ServicerContext):
```

Comment:
nit: Missed returning type annotation. Same thing apply to other functions in this PR, e.g., `_handle_sigint`, `_MethodHandle::stop`, `__init__`.